network.c

```diff
@@ -707,7 +732,11 @@ static struct iio_context * network_create_context(const struct iio_context_para
 
 	fd = create_socket(res, params->timeout_ms);
 	if (fd < 0) {
+#ifdef _WIN32
```

Comment:
I would rather have a stub implementation of `translate_wsa_error_to_posix()` for when __WIN32 is not defined